### ISSUE TITLE
Feature request. Hehe

### ISSUE THREAD

**Issue by aleckarlo** [mlab817/q-pipol#47]
(1) Can the "reviewer" role be able to search by Operating Unit?

(2) Can an indexing/auto-suggest feature be added in the search functions? Hehe

(3) From the "Add project" screen, can an "are you sure?" notice pop up when trying to leave the page?

**Comment by mlab817** [comment]
> (1) Can the "reviewer" role be able to search by Operating Unit?

Clarification on "searching operating unit". Is this on the view projects list? Or would you like reviewers to assign themselves?

> (2) Can an indexing/auto-suggest feature be added in the search functions? Hehe

Did you mean a search history feature?

**Comment by mlab817** [comment]
> (3) From the "Add project" screen, can an "are you sure?" notice pop up when trying to leave the page?

Fixed in 3ebf07b

Note however that this will only block navigating away from the page but will not prevent user refreshing the page.

**Comment by aleckarlo** [comment]
> Clarification on "searching operating unit". Is this on the view projects list? Or would you like reviewers to assign themselves?

Yep, I'm pertaining to the view projects list. I saw that the PAP Title is automatically tagged with the Operating Unit name upon saving the Project. I just think it would be easier for reviewers if the search box also returns that tagged Operating Unit name in the search bar so that you don't need to fill in the Title itself or a part of it. I believe it's like being able to filter projects per operating na rin kasi.

> Did you mean a search history feature?

Oh I meant an autocomplete or predictive search feature.

**Comment by mlab817** [comment]
On including operating unit in the search for projects, will do. However, this is not ideal since the search feature is currently limited to 10 search results only and it's really designed to search in the project titles.

I'll come up with an alternative view projects page instead which will allow filtering of projects based on operating units. probably a sidebar for OUs or something. If you can show me a good example, instead, I would love to see one.

I also think that the search function is not good.
